SIGS_AND_WGS.md

```diff
@@ -0,0 +1,28 @@
+# SIGs and WGs
+
+## SIG
+
+A **SIG (Special Interest Group)** is a group of people that owns a broad subject within the project - it has a scope that spans all aspects related to it (i.e. testing, structure, building etc).
+
+It maintains or supports processes (i.e. creation of output artifacts) and handles all topics touching it's subject, i.e. issues and design proposals.
+
+It defines mission and scope via it's charter, ownership of code via [OWNERS](https://www.kubernetes.dev/docs/guide/owners/), and defines its members roles (chairs and team leads) in [sigs.yaml].
+
+Examples:
+* [sig-network - charter](./sig-network/charter.md)
+* [sig-network - kubevirt/kubevirt OWNERS_ALIASES](https://github.com/kubevirt/kubevirt/blob/a7e0311d8704663351abd4bc9bbc8511753d2838/OWNERS_ALIASES#L60)
+* [sig-ci - sigs.yaml](https://github.com/kubevirt/community/blob/4f63a79c0ed810aa332cd6716d4986001d28bcd7/sigs.yaml#L119)
+
+## WG
+
+A **WG (Working Group)** is a group of people that owns a more specific topic within the project, possibly touching the scope of more than one SIG, with whom the WG interacts.
```

Comment:
I'm against redefining the k8s meaning of a WG here by suggesting that ownership is possible outside of a SIG [1][2].

As I've suggested elsewhere [3] we can avoid this by just using subprojects within a SIG [4] and keep the k8s definition of a WG to be a multi SIG collaboration vehicle. 

[1] https://github.com/kubernetes/community/blob/master/governance.md#working-groups
[2] https://github.com/kubernetes/community/blob/master/committee-steering/governance/wg-governance.md#is-it-a-working-group-yes-if
[3] https://github.com/kubevirt/community/pull/299#issuecomment-2149750036
[4] https://github.com/kubernetes/community/blob/master/governance.md#subprojects

Reply:
@lyarwood in Kubernetes WGs are short-lived, and I don't think that this serves node architecture related matters well (for which the initial PR was created), since for that matter clearly an ownership needs to exist in the longer term.

Therefore I have refrained from copying the k8s definitions.

Aside of that I am not talking about specific code ownership here (I agree that this is restricted to SIGs IMHO), but rather about topic ownership. Maybe since I am not a native speaker this is just a matter of words, so I could use some advice about how to rephrase it.

Reply:
@dhiller processes and deliverables are all also ultimately owned by a SIG within k8s thus my suggestion again to use a SIG sub project instead of redefining the k8s WG definition. These sub projects also have leads [1] and owners [2] so should accommodate what you want to achieve in  https://github.com/kubevirt/community/pull/298. I'm working through an example for instance types under SIG Compute within https://github.com/kubevirt/kubevirt/pull/12062 and https://github.com/kubevirt/community/pull/302 FWIW.

Reply:
First of all: what is the problem with redefining WGs for KubeVirt? 

Second: Actually, when looking at the document you provided, the sense of the above sentence isn't even redefining it, but rather just avoiding the phrase "short-lived", no?

In that way, and also looking at the document, I think that working group is perfectly right for node architecture, since it actually spans multiple SIGs.

And the sentence explicitly says "owns a more specific topic" and says nothing about code ownership, so again - what is wrong with that sentence?

Please help, thanks :)

Reply:
> First of all: what is the problem with redefining WGs for KubeVirt?

I just don't understand why it's required when the existing and well understood (at least across CNCF) definitions of k8s provide everything we need IMHO.

> Second: Actually, when looking at the document you provided, the sense of the above sentence isn't even redefining it, but rather just avoiding the phrase "short-lived", no?

That and suggesting ownership of a topic instead of being targeted at resolving a specific problem. 

> In that way, and also looking at the document, I think that working group is perfectly right for node architecture, since it actually spans multiple SIGs.

They definitely are for organising around a defined problem like initially standing up a new architecture across multiple SIGs yes but the on-going ownership of the topic, processes and deliverables should still reside in a SIG IMHO with specific architectures listed as sub projects.

> And the sentence explicitly says "owns a more specific topic" and says nothing about code ownership, so again - what is wrong with that sentence?

As I tried to say before SIGs own more than just code and WGs in k8s own nothing.

> Please help, thanks :)

FWIW I've not received any positive feedback on my stance here or in other PRs so it's likely that I'm standing alone on this hill.